### ISSUE TITLE
Passing ExistingWorkPolicy values to existingWorkPolicy parameter causes exception

### ISSUE THREAD

**Issue by indeednt** [vrtdev/flutter_workmanager#9]
```
Workmanager.initialize(callbackDispatcher, isInDebugMode: true);
Workmanager.registerPeriodicTask(
    "1",  
    "simpleTask", 
     existingWorkPolicy: ExistingWorkPolicy.replace, 
     frequency: Duration(hours: 24) ,
     initialDelay: Duration(seconds: 5), 
);
```

```
E/flutter (23220): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: Invalid argument: Instance of 'ExistingWorkPolicy'
E/flutter (23220): #0      StandardMessageCodec.writeValue (package:flutter/src/services/message_codecs.dart:387:7)
E/flutter (23220): #1      StandardMessageCodec.writeValue.<anonymous closure> (package:flutter/src/services/message_codecs.dart:384:9)
E/flutter (23220): #2      __InternalLinkedHashMap&_HashVMBase&MapMixin&_LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:367:8)
E/flutter (23220): #3      StandardMessageCodec.writeValue (package:flutter/src/services/message_codecs.dart:382:13)
E/flutter (23220): #4      StandardMethodCodec.encodeMethodCall (package:flutter/src/services/message_codecs.dart:519:18)
E/flutter (23220): #5      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:311:13)
E/flutter (23220): <asynchronous suspension>
E/flutter (23220): #6      Workmanager._register (package:workmanager/workmanager.dart:216:37)
E/flutter (23220): <asynchronous suspension>
E/flutter (23220): #7      Workmanager.registerPeriodicTask (package:workmanager/workmanager.dart:189:13)
E/flutter (23220): <asynchronous suspension>
E/flutter (23220): #8      HazizzNotification.scheduleNotificationAlarmManager (package:mobile/notification/notification.dart:156:17)
E/flutter (23220): #9      _AsyncAwaitCompleter.start (dart:async-patch/async_patch.dart:49:6)
E/flutter (23220): #10     HazizzNotification.scheduleNotificationAlarmManager (package:mobile/notification/notification.dart:142:49)
E/flutter (23220): #11     _SettingsPage.build.<anonymous closure> (package:mobile/pages/settings_page.dart:255:38)
E/flutter (23220): #12     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:77:64)
E/flutter (23220): #13     _rootRunUnary (dart:async/zone.dart:1132:38)
E/flutter (23220): #14     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
E/flutter (23220): #15     _FutureListener.handleValue (dart:async/future_impl.dart:126:18)
E/flutter (23220): #16     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:639:45)
E/flutter (23220): #17     Future._propagateToListeners (dart:async/future_impl.dart:668:32)
E/flutter (23220): #18     Future._complete (dart:async/future_impl.dart:473:7)
E/flutter (23220): #19     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
E/flutter (23220): #20     _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:28:18)
E/flutter (23220): #21     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:294:13)
E/flutter (23220): #22     showTimePicker (package:flutter/src/material/time_picker.dart)
E/flutter (23220): #23     _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:77:64)
E/flutter (23220): #24     _rootRunUnary (dart:async/zone.dart:1132:38)
E/flutter (23220): #25     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
E/flutter (23220): #26     _FutureListener.handleValue (dart:async/future_impl.dart:126:18)
E/flutter (23220): #27     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:639:45)
E/flutter (23220): #28     Future._propagateToListeners (dart:async/future_impl.dart:668:32)
E/flutter (23220): #29     Future._completeWithValue (dart:async/future_impl.dart:483:5)
E/flutter (23220): #30     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:513:7)
E/flutter (23220): #31     _rootRun (dart:async/zone.dart:1124:13)
E/flutter (23220): #32     _CustomZone.run (dart:async/zone.dart:1021:19)
E/flutter (23220): #33     _CustomZone.runGuarded (dart:async/zone.dart:923:7)
E/flutter (23220): #34     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:963:23)
E/flutter (23220): #35     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
E/flutter (23220): #36     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
```

**Comment by timrijckaert** [comment]
Hi 👋, 

I could produce the error on our end too.

**Comment by timrijckaert** [comment]
I just released version `0.0.10` of the plugin which addresses these issues.
Could you confirm it works on your end?

Sorry for the inconvenience.

**Comment by indeednt** [comment]
> I just released version `0.0.10` of the plugin which addresses these issues.
> Could you confirm it works on your end?
> 
> Sorry for the inconvenience.

It is working properly. Thanks for the quick fix. 😄
